### PR TITLE
feat(web): bOAT-1166

### PR DESCRIPTION
**Pull request by kandcdev**
## Describe your changes

add ability to change start date

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "clean": "rimraf .turbo node_modules dist",
     "clean:dev": "rimraf dist src/server/static/scripts src/server/static/styles",
     "dev": "cross-env NODE_ENV=local npm-run-all --silent clean:dev sass rollup --parallel --race watch:** nodemon",
-    "lint:js": "npx eslint .",
+		"lint:js": "npx eslint .",
```
Comment (james-hill-kc) [inline appeals/web/package.json]: Not a problem as this won't break anything, but was this indentation added by accident?

File: appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js [hunk 2]
```diff
@@ -13,27 +13,51 @@ const appealDataWithoutStartDate = {
...
 	});
 
-	describe('GET /start-case/confirmation', () => {
+	describe('GET /start-case/add/confirmation', () => {
 		it('should render "Case started" confirmation page', async () => {
-			const response = await request.get(`${baseUrl}/1/start-case/confirmation`);
+			nock('http://test/').get('/appeals/1').reply(200, appealDataWithoutStartDate);
+			const response = await request.get(`${baseUrl}/1/start-case/add/confirmation`);
+			const element = parseHtml(response.text);
+			expect(element.innerHTML).toMatchSnapshot();
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js:38]: Just a suggestion, you could add some quick checks for content you expect to be present in the snapshot, to help avoid regressions - there's some examples in `appeals/web/src/server/appeals/appeal-details/manage-linked-appeals/__tests__/manage-linked-appeals.test.js` - there's a tech debt ticket (https://pins-ds.atlassian.net/browse/BOAT-1407) which covers updating existing instances where this is missing, but I think we should probably try to add these for any new tests like this one

File: appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js [hunk 3]
```diff
@@ -13,27 +13,51 @@ const appealDataWithoutStartDate = {
...
+			nock('http://test/').post(`/appeals/1/appeal-timetables`).reply(200, {});
+			const response = await request.post(`${baseUrl}/1/start-case/change`).send({});
+			expect(response.statusCode).toBe(302);
+		});
+	});
+
+	describe('GET /start-case/change/confirmation', () => {
+		it('should render "Start date changed" confirmation page', async () => {
+			nock('http://test/').get('/appeals/1').reply(200, appealData);
+			const response = await request.get(`${baseUrl}/1/start-case/change/confirmation`);
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js:60]: As above, you could add checks for specific content in the snapshot here

File: appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js [hunk 4]
```diff
@@ -60,11 +67,83 @@ export const postStartDate = async (request, response) => {
...
+export const postChangeDate = async (request, response) => {
+	try {
+		const { appealId, documentationSummary } = request.currentAppeal;
+
+		if (documentationSummary?.lpaQuestionnaire?.status !== 'not_received') {
+			return response.render('app/500.njk');
+		}
+
+		const today = new Date();
+		const todayDayMonthYear = {
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js:116]: You can use the `dateToDayMonthYear` function in `lib/dates.js` to do this

**Review by james-hill-kc — COMMENTED** [review]
Looks good overall, just a few comments from me

File: appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js (hunk 4 above)
Comment (kandcdev) [reply appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js:116]: Hmm, i don't see one in there. Can you please check again?

File: appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js (hunk 2 above)
Comment (kandcdev) [reply appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js:38]: done

File: appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js (hunk 3 above)
Comment (kandcdev) [reply appeals/web/src/server/appeals/appeal-details/start-case/__tests__/start-case.test.js:65]: done

File: appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js (hunk 4 above)
Comment (kandcdev) [reply appeals/web/src/server/appeals/appeal-details/start-case/start-case.controller.js]: i see it now after update. done

**Review by james-hill-kc — APPROVED** [review]
LGTM!

**Pull request merged**